pkgs/applications/networking/instant-messengers/nheko/default.nix

```diff
@@ -42,6 +43,19 @@ stdenv.mkDerivation rec {
     hash = "sha256-2daXxTbpSUlig47y901JOkWRxbZGH4qrvNMepJbvS3o=";
   };
 
+  patches = [
+    (fetchpatch {
+      name = "nheko-fmt10.patch";
+      url = "https://gitlab.archlinux.org/archlinux/packaging/packages/nheko/-/raw/1b0d5c9eff6409dfd82953f346546d36c288a4a9/nheko-0.11.3-fix-for-fmt-10.patch";
```

Comment:
Is there an upstream report of this patch?

Reply:
This is also fixed upstream by https://github.com/Nheko-Reborn/nheko/pull/1552, but our version needs this larger patch.

Reply:
Is the archlinux patch a backport of this upstream patch:

https://github.com/Nheko-Reborn/nheko/commit/e89e65dc17020772eb057414b4f0c5d6f4ad98d0

?

Reply:
I missed that one! But indeed it does not apply to `0.11.3`.